Comments.md

```diff
@@ -124,3 +124,65 @@ func loadLastCheckpoint() -> GameState? {
     return loadedState
 }
 ```
+
+## Convention: Single line and multi line comment types
+
+Single line and multi line comments in Swift can be written in a couple of different forms, some of which enable additional features such as markdown to provide richer documentation.
+
+## Rationale
+
+- Single line comments can be written using two slashes, `//`, or using three slashes, `///`. `///` comments behave the same as `//` comments except that they enable the use of markdown and allow Xcode to create Quick Help documentation for the code. For these reasons, prefer the use of `///` comments over `//` comments when documenting public properties or methods.
+- Multi line comments can be written using `/* */` or `/** */`. `/** */` comments behave the same as `/* */` except that they enable the use of markdown and allow Xcode to create Quick Help documentation for the code. For these reasons, prefer the use of `/** */` comments over `/* */` comments when documenting public properties or methods.
```

Comment:
Xcode's documentation generation by default uses `///` (Editor->Structure->Add Documentation), and Xcode does recognizes that and display those documentation in auto-complete while the `/** */` documentation does not display the documentation in auto-complete. Do we want to recommend one over the other because QuickHelp markdown is preferred over autocomplete documentation? 
From a search in Swift's official guidelines, it looks like they prefer /// over /** */ for documentation: https://www.swift.org/documentation/api-design-guidelines/#fundamentals